src/Tags/Concerns/QueriesScopes.php

```diff
@@ -23,6 +23,10 @@ protected function parseQueryScopes()
     {
         $scopes = Arr::getFirst($this->params, ['query_scope', 'filter']);
 
+        if ($scopes === null) {
```

Comment:
can we pass `collect()` in as the default to line 24 above, for less code?

Reply:
Was able to clean this up by using an empty string as the default.